test/annexB/built-ins/String/prototype/trimRight/name.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-string.prototype.trimRight
+description: >
+  String.prototype.trimRight.name is "trimRight".
+info: >
+  String.prototype.trimRight ( )
+
+  17 ECMAScript Standard Built-in Objects:
+    Every built-in Function object, including constructors, that is not
+    identified as an anonymous function has a name property whose value
+    is a String.
+
+    Unless otherwise specified, the name property of a built-in Function
+    object, if it exists, has the attributes { [[Writable]]: false,
+    [[Enumerable]]: false, [[Configurable]]: true }.
+includes: [propertyHelper.js]
+features: [string-trimming]
+---*/
+
+verifyProperty(String.prototype.trimRight, "name", {
+  value: "trimRight",
```

Comment:
Same here.

Reply:
This is annex B ;)

Reply:
Yes, `trimLeft` and `trimRight` are Annex B features, but the tests should still match the specified behavior.

Reply:
Yep, I see the oversight now, thanks for your patience :)